com.unity.netcode.gameobjects/Tests/Runtime/MultiInstanceHelpers.cs

```diff
@@ -131,15 +126,16 @@ public static void Destroy()
             // Destroy the network manager instances
             foreach (var networkManager in NetworkManagerInstances)
             {
-                Object.Destroy(networkManager.gameObject);
+                Object.DestroyImmediate(networkManager.gameObject);
```

Comment:
Did we mean to make this change to `DestroyImmediate` here and below?

Reply:
Yep!

Reply:
So this is wrong :P we should not be using `DestroyImmediate` outside editor code as recommended by the Unity Manual